combat-trainer.lic

```diff
@@ -2130,11 +2337,11 @@ class TrainerProcess
 
   def astrology(game_state)
     return if game_state.retreating?
-
     retreat
-    bput('predict weather', 'You predict that', 'You are far too', 'you lack the skill to grasp them fully')
-    waitrt?
-    fput('engage')
+	check_heavens
+	  
+	bput("wield #{game_state.weapon_name}", 'You wield', 'You draw')
```

Comment:
We should use combat-trainer to wield the weapon
Examples
https://github.com/rpherbig/dr-scripts/blob/e4bd5ee16d20aecc6a1d3546dfdf4c87b488f92a/combat-trainer.lic#L240
https://github.com/rpherbig/dr-scripts/blob/e4bd5ee16d20aecc6a1d3546dfdf4c87b488f92a/combat-trainer.lic#L621
https://github.com/rpherbig/dr-scripts/blob/e4bd5ee16d20aecc6a1d3546dfdf4c87b488f92a/combat-trainer.lic#L1163

Reply:
do we need to worry about summoned weapons?

Reply:
Good call. Yes

Reply:
ok I will still from the bundling rope section

Reply:
appropriate